DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -33,7 +51,7 @@
     <a class="nhsuk-button left-button-mobile-margin-bottom nhsuk-u-margin-right-2" asp-controller="MyAccount" asp-action="EditDetails" asp-route-dlsSubApplication="@Model.DlsSubApplication.UrlSegment" role="button">
       Edit details
     </a>
-    <a class="nhsuk-button nhsuk-button--secondary nhsuk-u-margin-right-2" asp-controller="ChangePassword" asp-action="Index" asp-route-dlsSubApplication="@Model.DlsSubApplication.UrlSegment" role="button">
+    <a class="nhsuk-button left-button-mobile-margin-bottom nhsuk-button--secondary nhsuk-u-margin-right-2" asp-controller="ChangePassword" asp-action="Index" asp-route-dlsSubApplication="@Model.DlsSubApplication.UrlSegment" role="button">

```

Comment:
Note: adding this class makes the whitespace between these buttons better on mobile